src/main/webapp/scripts/calendar-script.js

```diff
@@ -0,0 +1,190 @@
+import {updateBar} from './nav-bar.js';
+import {initPromise} from './account-info.js';
+
+
+initPromise.then(function() {
+  const auth2 = gapi.auth2.getAuthInstance();
+  updatePage();
+  auth2.isSignedIn.listen(updatePage);
+  gapi.client.load('calendar', 'v3', function() {
+    getCalendarEvents('c_96sboq7a00dhtc9c24pr7mpi3o@group.calendar.google.com', false)
+        .then(function(events) {
+      if (events.length > 0) {
+        for (let i = 0; i < events.length; i++) {
+          const event = events[i];
+          addEventToPage(event);
+        }
+      }
+    });
+  });
+});
+
+/**
+ * Gets events from a calendar with specified ID
+ * @param {String} id ID of calendar we want events from
+ * @param {Boolean} deleted Whether or not we want to include deleted events
+ * @return {Promise} Promise which resolves after the events of the calendar are listed
+ */
+function getCalendarEvents(id, deleted) {
+  return gapi.client.calendar.events.list({
+    'calendarId': id,
+    'timeMin': (new Date()).toISOString(),
+    'showDeleted': deleted,
+    'singleEvents': true,
+    'orderBy': 'startTime',
+  }).then(function(response) {
+    const events = response.result.items;
+    return events;
+  });
+}
+
+/**
+ * Adds an event to the webpage
+ * @param {Event} event Event to add
+ */
+function addEventToPage(event) {
```

Comment:
where do events get added bottom of list or top on page

Reply:
in the ul

Reply:
should it be organized by like time like soonest to latest

Reply:
Ya it is ordered in that way